### ISSUE TITLE
这段为啥要重复添加一次？

### ISSUE THREAD

**Issue by Sokkafan** [bubbliiiing/mask-rcnn-tf2#35]
#------------------------------------------------------------------#
#获得RPN网络的预测结果，进行格式调整，把五个特征层的结果进行堆叠
#------------------------------------------------------------------#
    rpn_class_logits, rpn_class, rpn_bbox = [],[],[]
    for p in rpn_feature_maps:
        logits,classes,bbox = rpn([p])
        rpn_class_logits.append(logits)
        rpn_class.append(classes)
        rpn_bbox.append(bbox)
        
#------------------------------------------------------------------#
#此时获得的rpn_class_logits、rpn_class、rpn_bbox的维度是
#rpn_class_logits    : Batch_size, num_anchors, 2
#rpn_class           : Batch_size, num_anchors, 2
#rpn_bbox            : Batch_size, num_anchors, 4
#------------------------------------------------------------------#
for p in rpn_feature_maps:
    logits,classes,bbox = rpn([p])
    rpn_class_logits.append(logits)
    rpn_class.append(classes)
    rpn_bbox.append(bbox)

**Comment by Sokkafan** [comment]
之前出现过 #26 类似的问题，把这段重复的删掉以后就好了。 @bubbliiiing

**Comment by bubbliiiing** [comment]
好像是错了，我的问题………………我改一下
